frontend/app/constants.ts

```diff
@@ -1,4 +1,5 @@
 export const constants = {
+  DOMAIN: 'https://universalgrave.com',
```

Comment:
@democrazyfi wouldnt this break down on testnet since URL wouldnt match?

Reply:
used networkConfig instead 👍